src/imeta.cpp

```diff
@@ -1214,6 +1226,7 @@ parse_program_options(int _argc,
         ("help,h",                                             "Show imeta help")
         ("verbose,v",                                          "Verbose output")
         ("very_verbose,V",                                     "Very verbose output")
+        ("as_admin,M",                                         "Execute as administrator")
```

Comment:
Would it be better to use "admin_mode" instead of "as_admin" for consistency?

Reply:
sure. will change it.

Reply:
done